srcpkgs/ibus-array/template

```diff
@@ -1,19 +1,13 @@
 # Template file for 'ibus-array'
 pkgname=ibus-array
-version=0.2.2
+version=0.2.3
 revision=1
-build_style=gnu-configure
-hostmakedepends="automake pkg-config python3 gettext-devel libtool"
-makedepends="ibus-devel gettext-devel sqlite-devel"
+build_style=meson
+hostmakedepends="pkg-config"
```

Comment:
```suggestion
hostmakedepends="pkg-config python3 gettext"
```
Keep:

- python3, just in case we use anything other than meson for meson, or meson go away from python
- gettext for translation